utils/sn_flux_est.py

```diff
@@ -25,6 +25,7 @@
 import mwa_metadb_utils
 import submit_to_database
 import process_vcs
+import prof_utils
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused import prof_utils](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4645369)